[OSDOCS#9302]: 4.15 RN: Fix wrong acronyms (#70813)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

[id="ocp-4-15-CAPO-tech-preview-no-upgrade"]
==== CAPO integration into the cluster CAPI Operator (Tech Preview)

> If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates `Cluster` and `OpenStackCluster` resources for the current {product-title} cluster.

Reviewer: is lifestyle correct here? lifecycle seems like a more appropriate word in this context.

Author: Oh yes. Thanks you.

> If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates an additional OpenStack cluster and an additional {product-title} cluster for the current {product-title} cluster.

Reviewer 2: ```suggestion
If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates `Cluster` and `OpenStackCluster` resources for the current {product-title} cluster.
```

Reviewer 2: Note that the `Cluster` and `OpenStackCluster` here are specific API resources provided by CAPI and CAPO respectively. We're *not* talking about "clusters" in the generic sense. As such, I think they need to be formatted as `literals`.

> It is now possible to configure the CAPI machine and OpenStack machine resources in a similar fashion to how the Machine API (MAPI) resources are configured. It is important to note that the CAPI resources are equivalent to MAPI resources and not identical.

Reviewer 2: ```suggestion
It is now possible to configure CAPI `Machine` and CAPO `OpenStackMachine` resources in similar fashion to how Machine API (MAPI) resources are configured. It is important to note that the CAPI resources are equivalent to MAPI resources but not identical.
```
